Foodhut/src/sections/introduction.html

```diff
@@ -19,12 +19,10 @@ <h1 class="header-introduction">
       Best cooks and best delivery guys all at your service. Hot tasty food will reach you in 60 minutes.
     </p>
     <form class="search-box-container flex between">
-      <div class="search-black-icon">
-        <img src="./assets/images/search-icon-black.svg" alt="search black icon"/>
-      </div>
+      <div class="search-black-icon"></div>
       <input class="search-container" type="text" placeholder="Search food" />
       <button class="search-white-icon" type="button">
-        <img src="./assets/images/search-icon-white.svg" alt="search white icon"/>
+        <div class="search-icon"><img src="./assets/images/search-icon-white.svg" alt="search white icon"/></div>
```

Comment:
`alt` not good, what if icon change the color, in that case you must change the `alt` as well right?

Reply:
i changed it, thanks

Reply:
Not good solution, pls try again with good alt content

Reply:
i changed it , thanks